
From: Mike Miller <mike.miller@hp.com>

This patch fixes the vendor ID for our cciss based SATA controller due out
later this year and adds the new PCI ID to pci_ids.h.

Also changes DRIVER_NAME to HP CCISS.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/cciss.c   |    6 +++---
 25-akpm/include/linux/pci_ids.h |    1 +
 2 files changed, 4 insertions(+), 3 deletions(-)

diff -puN drivers/block/cciss.c~cciss-updates-id-change-for-v100-controller-for-268-rc3 drivers/block/cciss.c
--- 25/drivers/block/cciss.c~cciss-updates-id-change-for-v100-controller-for-268-rc3	Thu Aug  5 15:38:31 2004
+++ 25-akpm/drivers/block/cciss.c	Thu Aug  5 15:38:31 2004
@@ -46,14 +46,14 @@
 #include <linux/completion.h>
 
 #define CCISS_DRIVER_VERSION(maj,min,submin) ((maj<<16)|(min<<8)|(submin))
-#define DRIVER_NAME "Compaq CISS Driver (v 2.6.2)"
+#define DRIVER_NAME "HP CISS Driver (v 2.6.2)"
 #define DRIVER_VERSION CCISS_DRIVER_VERSION(2,6,2)
 
 /* Embedded module documentation macros - see modules.h */
 MODULE_AUTHOR("Hewlett-Packard Company");
 MODULE_DESCRIPTION("Driver for HP Controller SA5xxx SA6xxx version 2.6.2");
 MODULE_SUPPORTED_DEVICE("HP SA5i SA5i+ SA532 SA5300 SA5312 SA641 SA642 SA6400"
-			" SA6i");
+			" SA6i V100");
 MODULE_LICENSE("GPL");
 
 #include "cciss_cmd.h"
@@ -82,7 +82,7 @@ const struct pci_device_id cciss_pci_dev
 		0x0E11, 0x4091, 0, 0, 0},
 	{ PCI_VENDOR_ID_COMPAQ, PCI_DEVICE_ID_COMPAQ_CISSC,
 		0x0E11, 0x409E, 0, 0, 0},
-	{ PCI_VENDOR_ID_COMPAQ, PCI_DEVICE_ID_COMPAQ_CISSC,
+	{ PCI_VENDOR_ID_HP, PCI_DEVICE_ID_HP_CISS,
 		0x103C, 0x3211, 0, 0, 0},
 	{0,}
 };
diff -puN include/linux/pci_ids.h~cciss-updates-id-change-for-v100-controller-for-268-rc3 include/linux/pci_ids.h
--- 25/include/linux/pci_ids.h~cciss-updates-id-change-for-v100-controller-for-268-rc3	Thu Aug  5 15:38:31 2004
+++ 25-akpm/include/linux/pci_ids.h	Thu Aug  5 15:38:31 2004
@@ -675,6 +675,7 @@
 #define PCI_DEVICE_ID_HP_SX1000_IOC	0x127c
 #define PCI_DEVICE_ID_HP_DIVA_EVEREST	0x1282
 #define PCI_DEVICE_ID_HP_DIVA_AUX	0x1290
+#define PCI_DEVICE_ID_HP_CISS		0x3211
 
 #define PCI_VENDOR_ID_PCTECH		0x1042
 #define PCI_DEVICE_ID_PCTECH_RZ1000	0x1000
_
